converter/bijection/example_test.go

```diff
@@ -0,0 +1,206 @@
+package bijection
+
+import (
+	"testing"
+
+	"github.com/Shopify/sarama"
+	"github.com/grafana/agent/component/common/config"
+	"github.com/grafana/agent/component/loki/source/kafka"
+	"github.com/grafana/agent/pkg/river/rivertypes"
+	"github.com/grafana/dskit/flagext"
+	"github.com/grafana/loki/clients/pkg/promtail/scrapeconfig"
+	common "github.com/prometheus/common/config"
+	"github.com/prometheus/common/model"
+)
+
+func TestKafkaBijection(t *testing.T) {
```

Comment:
This test verifies a two-way conversion of a more complex structure (kafka logs source config). 

The definition of this bijection (conversion logic) is below the test.